### ISSUE TITLE
closing laptop lid on deno run --watch causes "interrupted system call" error

### ISSUE THREAD

**Issue by andykais** [denoland/deno#10066]
interesting error that occurred when I shut my laptop lid and reopened it on a watched process

`deno run --unstable --watch file.ts`
```
Watcher Process finished! Restarting on file change...
thread '<unnamed>' panicked at 'poll failed: Os { code: 4, kind: Interrupted, message: "Interrupted system call" }', /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/notify-5.0.0-pre.5/src/inotify.rs:134:47
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

the watcher appears to stop working after this error is shown, but the process does not exit

**Comment by kitsonk** [comment]
The process should not panic and terminate.

**Comment by Liamolucko** [comment]
It looks like this was fixed in notify-rs/notify@50e90ea57f1984395f1f7172d100ef6505b3d84b, so #10058 should fix this.
